packages/attack-http-requests/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@dazn/chaos-squirrel-attack-http-requests",
+  "version": "0.0.1",
+  "description": "Chaos Squirrel attack to HTTP requests",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "shx rm -rf dist && tsc",
+    "build:watch": "tsc -w",
+    "test": "jest"
+  },
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/getndazn/chaos-squirrel.git"
+  },
+  "author": "Lou Bichard <louis.bichard@dazn.com>",
+  "license": "UNLICENSED",
```

```suggestion
  "license": "MIT",
```

I think you started this before we OSS'ed it!